Pixel Skater Dude

NotYourAveragePaladin opened the pull request:
# New Port for Pixel Skater Dude

## Game Information
- **Title**: Pixel Skater Dude
- **URL**: https://jmation.itch.io/pixel-skater-dude

## Submission Requirements

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

tabreturn commented:
Why are there Candle Prick files in this PR?

NotYourAveragePaladin commented:
Me and plaidman went into powershell and deleted them all. I have no idea
how they came back.

On Sat, Apr 12, 2025, 4:53 PM tabreturn ***@***.***> wrote:

> Why are there Candle Prick files in this PR?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799067838>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BO4CZU4XUX5LECVWTIGI3R32ZGKNJAVCNFSM6AAAAAB3AFJHPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJZGA3DOOBTHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> *tabreturn* left a comment (PortsMaster/PortMaster-New#1516)
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799067838>
>
> Why are there Candle Prick files in this PR?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799067838>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BO4CZU4XUX5LECVWTIGI3R32ZGKNJAVCNFSM6AAAAAB3AFJHPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJZGA3DOOBTHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

NotYourAveragePaladin commented:
We might have to do something else together. But he helped me restart my
tree and clean it

On Sat, Apr 12, 2025, 5:30 PM Dank Nugz801 ***@***.***> wrote:

> Me and plaidman went into powershell and deleted them all. I have no idea
> how they came back.
>
> On Sat, Apr 12, 2025, 4:53 PM tabreturn ***@***.***> wrote:
>
>> Why are there Candle Prick files in this PR?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799067838>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BO4CZU4XUX5LECVWTIGI3R32ZGKNJAVCNFSM6AAAAAB3AFJHPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJZGA3DOOBTHA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>> *tabreturn* left a comment (PortsMaster/PortMaster-New#1516)
>> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799067838>
>>
>> Why are there Candle Prick files in this PR?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799067838>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BO4CZU4XUX5LECVWTIGI3R32ZGKNJAVCNFSM6AAAAAB3AFJHPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJZGA3DOOBTHA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

plaidman commented:
haha we cleaned the tree for slime 3k. not this branch. we need to do this branch separately. and possibly other branches you already have open

NotYourAveragePaladin commented:
Oh boy. Well, I think deleted a bunch of branches I don't need. Can I
delete the tabteturn patch1 and jeodc patch¹ branches?

On Sat, Apr 12, 2025, 8:36 PM Jason Tomsic ***@***.***> wrote:

> haha we cleaned the tree for slime 3k. not this branch. we need to do this
> branch separately. and possibly other branches you already have open
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799557933>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BO4CZU42PENUZ6CZXHJQU3D2ZHEUHAVCNFSM6AAAAAB3AFJHPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJZGU2TOOJTGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> *plaidman* left a comment (PortsMaster/PortMaster-New#1516)
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799557933>
>
> haha we cleaned the tree for slime 3k. not this branch. we need to do this
> branch separately. and possibly other branches you already have open
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2799557933>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BO4CZU42PENUZ6CZXHJQU3D2ZHEUHAVCNFSM6AAAAAB3AFJHPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJZGU2TOOJTGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

tabreturn commented on ports/pixelskaterdude/README.md:
its retro style ...
-->
Its retro style ...

some issues i was having with it
-->
some issues I was having with it

What's "Left-Movement"?
|Left-Movement|Brake|

tabreturn commented on ports/pixelskaterdude/port.json:
- I dunno about  ["action", "platformer", "simulation"] -- Maybe just ["sports"], or ["sports", "action"] ?

- "name": "itch..io" --> "name": "itch.io",

Cebion commented on ports/pixelskaterdude/Pixel Skater Dude.sh:
```diff
@@ -0,0 +1,40 @@
+# CD and set permissions
+cd $GAMEDIR
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+$ESUDO chmod +x -R $GAMEDIR/*
```
This is not being done anymore.

NotYourAveragePaladin replied:
So change to this?
cd $GAMEDIR
> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1

NotYourAveragePaladin replied:
Or no... that's the same thing

NotYourAveragePaladin replied:
So just delete the 'esudo chmod' line 25 entirely?

ben-willmore commented:
This is a lot of changes for a PR. It would be much better to iron these out in the testing thread before PR because it's hard to review this.

NotYourAveragePaladin commented:
It's all been fixed

On Mon, Apr 14, 2025, 1:50 PM Ben Willmore ***@***.***> wrote:

> This is a lot of changes for a PR. It would be much better to iron these
> out in the testing thread before PR because it's hard to review this.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2802831900>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BO4CZU6DQRV5A3676LUPHZL2ZQGQ3AVCNFSM6AAAAAB3AFJHPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMBSHAZTCOJQGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> *ben-willmore* left a comment (PortsMaster/PortMaster-New#1516)
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2802831900>
>
> This is a lot of changes for a PR. It would be much better to iron these
> out in the testing thread before PR because it's hard to review this.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PortsMaster/PortMaster-New/pull/1516#issuecomment-2802831900>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BO4CZU6DQRV5A3676LUPHZL2ZQGQ3AVCNFSM6AAAAAB3AFJHPGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMBSHAZTCOJQGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
